pkg/agent/controller/networkpolicy/l7engine/reconciler.go

```diff
@@ -266,13 +274,40 @@ func (r *Reconciler) StartSuricataOnce() {
 	})
 }
 
+func (r *Reconciler) initializeL7Flows() error {
```

Comment:
@antoninbas Here is the latest implementation I pushed yesterday. I forgot mentioning it in the discussion.

Reply:
the logic looks good to me